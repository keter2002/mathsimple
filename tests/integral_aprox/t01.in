1 2 6 1/x
