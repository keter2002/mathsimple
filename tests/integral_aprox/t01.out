|1.000000|x/
1.000000 2.000000 6.000000 0.166667
RI - LI = -0.083333
RI: 0.653211
LI: 0.736544
MI: 0.692284
Trapezoidal Rule: 0.694877
Simpson Rule: 0.693170
